roles/kubernetes/master/tasks/main.yml

```diff
@@ -58,7 +58,7 @@
   command: "{{ bin_dir }}/kubectl get ns kube-system"
   register: 'kubesystem'
   changed_when: False
-  ignore_errors: yes
+  failed_when: False
```

Comment:
I'm guessing it does since I get the following if I try to run it manually once the stack is up:

```
root@k8s-01:~# kubectl create -f /etc/kubernetes/kube-system-ns.yml
Error from server: error when creating "/etc/kubernetes/kube-system-ns.yml": namespaces "kube-system" already exists
```